testing-guidance/workflow-tests/shapes/shapes-lkg-workflows.md

```diff
@@ -0,0 +1,71 @@
+# Shapes LKG Workflow Tests
+
+## Background
+
+### Dependent Components
+A lot of other components depend on shape components to define geometry, for example:
+
+- PhysX Shape Collider
+- Recast Navigation PhysX Provider
+- Terrain Height Gradient List
+- Terrain Layer Spawner
+- Terrain Macro Material
+- Terrain Physics Heightfield Collider
+- Terrain Surface Materials List
+- Vegetation Altitude Filter
+- Vegetation Layer Spawner
+- Vegetation Layer Blocker
+- Vegetation Shape Intersection Filter
+- Diffuse Probe Grid
+- Light
+- Reflection Probe
+
+### PhysX Shape Collider Component
+The following shape components are supported by the PhysX Shape Collider component:
+
+- Box
+- Capsule
+- Sphere
+- Cylinder
+- Polygon Prism
+- Quad
+
+Polygon Prism is special because it might not be convex, and is automatically split into convex parts for use with the PhysX Shape Collider component. Note that the PhysX Shape Collider component does not currently support modifying shapes at runtime.
+
+### Translation Offsets
+The following shapes support translation offsets (currently behind a feature flag - set "EnableShapeComponentTranslationOffset": true under "Amazon" -> "Preferences"):
+
+- Axis Aligned Bounding Box
+- Box
+- Capsule
+- Sphere
+
+### Component Modes
+The following shapes have component modes:
+
+- Axis Aligned Bounding Box
+- Box
+- Capsule
+- Sphere
+- Polygon Prism
+- Spline
+- Tube Shape
+- White Box
+
+### Non-uniform Scale
+The following shapes support non-uniform scale:
+- Box
+- Quad
+- Polygon Prism
+
+## Workflows
+### Area: Editing shapes
+**Docs:**
+- [Shape Components](https://www.o3de.org/docs/user-guide/components/reference/shape/)
+- [PhysX Shape Collider](https://www.o3de.org/docs/user-guide/components/reference/physx/shape-collider/)
+- [Non-uniform Scale Component](https://www.o3de.org/docs/user-guide/components/reference/non-uniform-scale/non-uniform-scale/)
+
+| Workflow | Requests | Things to Watch For |
```

Comment:
It's missing the Timebox with a rough estimation of how long it'd take to run the workflow with all the variations.

Reply:
I don't really know how long it would take to test everything. My expectation was that that information would be added as this document gets iterated on.

Reply:
Adding it in the future is fine by me.